src/OrchardCore.Modules/OrchardCore.Markdown/wwwroot/Scripts/simplemde.media.min.js

```diff
@@ -1 +1 @@
-!function(e){"use strict";e(document).on("simplemde:initialize",function(o,a){a.toolbar[9].action=function(o){console.log("custom drawImage"),e("#mediaApp").detach().appendTo("#mediaModalMarkdown .modal-body"),e("#mediaApp").show();e("#mediaModalMarkdown").modal();e("#mediaSelectButton").on("click",function(a){o.codemirror.replaceSelection('{{ "'+mediaApp.selectedMedia.mediaPath+'" | asset_url | img_tag }}'),e("#mediaModalMarkdown").modal("hide")})}})}(jQuery);
```

Comment:
It didn't change, you should check the diffs before committing, and if there is no change just ignore this file. Please check other files

Reply:
Done. I ignored all but Workflow module files.